docs/samples/browser-plugin-meetings/index.html

```diff
@@ -596,6 +596,8 @@ <h2 class="collapsible">Meeting Streams</h2>
           <div>
             <fieldset>
               <legend>Breakout Sessions</legend>
+              <input type="checkbox" id="enable-breakout" onclick="toggleBreakout()">
+              <label for="enable-breakout">Toggle breakout</label><br>
```

Comment:
nitpick: "Enable breakout" might be a better text - it would be more clear what it means when the checkbox is ticked or unticked

Reply:
got it, thanks